docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding.md

```diff
@@ -0,0 +1,146 @@
+5. The merchant receives access to the PSP Back Office, where data regarding commissions and payments can be checked.
+
+{% info_block infoBox %}
+
+For more information about the payment setup and available options, ask your Spryker contact.
+
+{% endinfo_block %}
+
+### 6) Operator: Set up the order management process
+
+The State Machine models the order management process, including both delivery and return parts that can be defined per merchant. It helps if you have big and small merchants with different fulfillment processes. Otherwise, you can use one State Machine flow for all your merchants.
+
+
+There are two fulfillment models applicable to Marketplace you can choose:
+* Fulfillment by merchants: Merchants receive, process, and dispatch orders directly to your marketplace buyers.
+* Fulfillment by marketplace: Merchants sell products and the operator manages the logistics, including returns.
+
+The shipping process needs to be set up on a project level.
+
+The State Machine can be set up to route an automated email to the relevant merchant once an order is created.
+
+The following diagram shows an example of a simple State Machine workflow, where every merchant handles their own shipments and returns.
+
+![state-machine-workflow-example](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-6-state-machine-workflow-example.png)
+
+### 7) Merchant and operator: Approve products and offers and activate the merchant
+
+The following steps are taken by the operator:
+
+1. Do a final check of the merchant's public profile, products, and offer quality. 
+2. [Activate merchants](/docs/pbc/all/merchant-management/{{site.version}}/marketplace/manage-in-the-back-office/manage-merchants.html#activating-and-deactivating-merchants) and [approve products](/docs/marketplace/user/merchant-portal-user-guides/{{site.version}}/products/concrete-products/managing-marketplace-concrete-product.html#activating-and-deactivating-a-concrete-product) and [offers](/docs/marketplace/user/back-office-user-guides/{{site.version}}/marketplace/offers/managing-merchant-product-offers.html#approving-or-denying-offers) in the Back Office. 
+3. Optional: Activate and approve products and offers through the data importer.
+4. Optional: To optimize the product and offer approval process, define specific rules per merchant. 
+
+{% info_block infoBox %}
+
+Trusted merchants can have an automatic pre-approval, while new merchants have their new product listings pending first.
+
+{% endinfo_block %}
+
+![offers-in-the-back-office](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-7-offers-in-the-back-office.png)
+
+![offers-on-the-storefront](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-7-offers-on-storefront.png)
```

Comment:
let's change this image from the non-existing Storefront with the public one from https://www.de.b2c-marketplace.demo-spryker.com/en/merchant/budget-cameras. So we can take the part on top with image, description, etc. without legal part.

Reply:
```suggestion
![offers-on-the-storefront](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-7-offers-on-the-storefront.png)
```